test/test_utils.py

```diff
@@ -225,3 +225,34 @@ def test_invalid_png_data(self):
                           imageinfo.identify, data)
         self.assertRaises(imageinfo.UnrecognizedFormat,
                           imageinfo.identify, data)
+
+
+class TestEncodeFilename(unittest.TestCase):
+
+    @unittest.skipIf(os.path.supports_unicode_filenames , "unicode is supported")
+    def test_EncodeFilenames_Nosupport(self):
+        self.assertEqual(b'\xe5\xb0\x8f\xe5\xae\xa4', util.encode_filename("小室"))
+        self.assertEqual(b'\xe5\xb0\x8f', util.encode_filename('小'))
+
+    @unittest.skipUnless(os.path.supports_unicode_filenames and (sys.platform == 'darwin'), "requires unicode support and darwin os")
```

Comment:
Extra parenthesis are useless here: `os.path.supports_unicode_filenames and sys.platform == 'darwin'`